lib/structures/Member.js

```diff
@@ -26,7 +26,8 @@ const VoiceState = require("./VoiceState");
 * @prop {String?} nick The server nickname of the member
 * @prop {String[]} roles An array of role IDs this member is a part of
 * @prop {User} user The user object of the member
-* @prop {Permission} permission The guild-wide permissions of the member
+* @prop {Permission} permission The guild-wide permissions of the member. Using this property is deprecated and will not be supported in future versions.
+* @prop {Permission} permissions The guild-wide permissions of the member
```

Comment:
Pretty sure you meant to make this prop `{Permissions}`, and not the same as the above (that this PR would deprecate). Right now both props are the same.

Reply:
????????????????????????????????????????

Reply:
Instead of making incorrect corrections to something that's already correct, please look at the documentation. Also I doubt I'm the only person who thinks this but please don't leave useless comments on PRs, thanks :)

Reply:
@bsian03 Explain how this is incorrect? You have two props that are the exact same- `{Permission}`. Isn't the **entire** purpose of this PR to change `Member#Permission` to `Member#Permissions`, therefore the added deprecated text should be on the `Member#Permission` prop for the docs, and the new one should be `Member#Permissions`, as plural.

Reply:
As it is currently, If I'm not mistaken, you'd have `Member#Permission` on the docs twice. One with the added deprecated text, and one without. The change I'm suggesting is that you change the new, non-deprecated point to `Permissions`, making the non-deprecated point plural for the docs. Pretty sure that's the entire point of this PR.

Reply:
Lmfao actually, ignore those comments just made- Was on mobile earlier, and after looking at the file properly I see that's just the type, not the name. Totally sorry for the mis-read haha